src/plugins/intel_cpu/src/nodes/transpose.h

```diff
@@ -38,6 +38,10 @@ class Transpose : public Node {
     bool needPrepareParams() const override;
     void prepareParams() override;
 
+    void setFakeTranspose(bool fake) {
```

Comment:
Lets align implementation and naming with Reorder: https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/nodes/reorder.h#L51
It also assumes we should skipe PrepareParams, CreatePrimtive stages and so on

Reply:
done, thanks